### PR TITLE
CAP-0042: add the diff header to the code snippet in order to highlight it

### PR DESCRIPTION
**Pull request by overcat**

### CODE REVIEW

**Comment by overcat** [comment]
I think you will improve it later, so I closed it. Do I need to add a `mddiffcheck.base` and submit it?

File: core/cap-0042.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Note that this CAP does not modify how transactions gets applied (ie: the apply
 
 ### XDR Changes
 
-```
+```diff
```
Comment (leighmcculloch) [inline core/cap-0042.md]: The needs the attribute that specified the base ref. E.g.
```suggestion
```diff mddiffcheck.base=74498070b99a7fb1d18b78d104f95d797b4f4c2c
```
@MonsieurNicolas can you confirm the right base ref for this diff?

**Review by leighmcculloch — COMMENTED** [review]
Thanks for catching the xdr diff want annotated as a diff! We need one change to tell mddiffcheck, and the reader, what ref the stuff is based on.

**Comment by overcat** [comment]
BTW, should [this](https://github.com/stellar/stellar-protocol/commit/5a1c8c9c00aee6656c9e80fcc7f3c6af2ed3b6a4#diff-4bd548be9848895160eb5d359c7fe1b1478c6c96c6a762c3ca0aafe1bf0e135cR155) be changed to `case 1:`

File: core/cap-0042.md (hunk 1 above)
Comment (MonsieurNicolas) [reply core/cap-0042.md]: I ended up opening https://github.com/stellar/stellar-protocol/pull/1099 : for whatever reason the diff was slightly off

**Comment by leighmcculloch** [comment]
@MonsieurNicolas fixed this up here: https://github.com/stellar/stellar-protocol/pull/1099

Thanks @overcat !

**Pull request closed without merging**